roles/ocp_netobserv/README.md

```diff
@@ -0,0 +1,126 @@
+netobserv_region                       | us-east-1                       | No          | Object Storage region
+netobserv_loki_size                    | 1x.extra-small                  | No          | Loki Stack size See [Sizing](https://docs.openshift.com/container-platform/4.13/logging/cluster-logging-loki.html#deployment-sizing_cluster-logging-loki)
+netobserv_storage_class                | managed-nfs-storage             | No          | Storage class for the Loki Stack
+
+## Role requirements
+  - An Storage Class
+  - Loki-operator
+  - Network observability operator
+  - Supported Log Store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation) credentials (access_key and access_key_secret)
+  - Logs Store endpoint address
+
+## Usage example
+
+See below for some examples of how to use the ocp_netobserv role to configure the Network Observability subsystem.
+
+Create the Netobserv stack with default values:
+
+```yaml
+- name: "Setup Netobserv stack"
+  vars:
+    ocp_netobserv_conf_file: path-to/netobserv-config.yml
+  ansible.builtin.include_role:
+    name: redhatci.ocp.ocp_netobserv
+```
+
+Use a config file
+```yaml
+- name: "Setup Netobserv stack"
+  vars:
+    ocp_netobserv_conf_file: path-to/netobserv-config.yml
+  ansible.builtin.include_role:
+    name: redhatci.ocp.ocp_netobserv
+```
+
+Sample of a configuration file:
+```yaml
+---
+netobserv_action: 'install'
+netobserv_sampling: 50
+netobserv_agent_privileged: true
+netobserv_agent_memory: 50Mi
+netobserv_agent_cpu: 100m
+netobserv_agent_limits_memory: 800Mi
+netobserv_processor_memory: 100Mi
+netobserv_processor_cpu: 100m
+netobserv_processor_limits_memory: 800Mi
+netobserv_console_avg_utilization: 50
+netobserv_console_max_replicas: 3
+netobserv_console_limits_memory: 100Mi
+netobserv_console_cpu: 100m
+netobserv_console_memory: 50Mi
+netobserv_loki_tls_insecure_skip_verify: true
+netobserv_access_key_id: minioadmin
+netobserv_access_key_secret: minioadmin
+netobserv_bucket: network
+netobserv_endpoint: http://minio-service.minio:9000
+netobserv_region: us-east-1
+netobserv_loki_size: 1x.extra-small
+netobserv_storage_class: "managed-nfs-storage"
+...
+
+## Validaton
+
+To confirm that the stack is working properly:
+1. Validate that a "Logs" entry has been added to the OCP console under the "Observe" menu.
+1. Verify that the Object Storage bucket has started receiving files.
+
+## Troubleshooting
+
+The following are some recommended actions in case there are issues during the deployment.
+
+1. Confirm that all the pods in the `netobserv` namespace are running.
+1. Confirm that the OCP cluster is able to reach the Object Storage bucket. Check the ingester pods for errors when flushing logs tables.
+    ```ShellSession
+    $ oc logs -n netboserv logging-loki-ingester-0 | grep flush
+    ```
+
+# References
```

Comment:
Should we also add references to how to take a look to network observability info in the OCP console?

Reply:
I had that part in the validation section, renamed it to `Access the Netobserv GUI`, and reworded the section.  

Sure, I will work on documenting how to access the OCP consoles in a separated change.